src/assets/scss/app.scss

```diff
@@ -62,3 +62,105 @@
 // Motion UI
 @include motion-ui-transitions;
 @include motion-ui-animations;
+
+$message-new: $white;
+$message: $body-background;
+
+.footer{
+    text-align: center;
+}
+
+.content{
+    width: 80%;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+    white-space: nowrap;
+}
+
+.event-container{
+    h4{
+        color: $custom-menu-color;
+        padding-left: 1rem;
+    }
+
+    .event-message{
+        padding:1rem;
```

Comment:
nebylo by čitelnější všechno uvádět v pixelech a převádět to až do výstupu přes funkci - můžeš využít funkci rem-calc.. Jinak prosím hlídej si formátování chybí mezera u vlastnosti..

Reply:
na toto se jeste kouknu